hr_timesheet_task_stage/views/account_analytic_line.xml

```diff
@@ -17,15 +17,15 @@
                     tabindex="-1"
                     type="object"
                     icon="fa-folder-o"
-                    attrs="{'invisible': ['|', ('is_task_closed', '=', True), ('task_id', '=', False)]}"
+                    invisible="is_task_closed == True or task_id == False"
                 />
                 <button
                     name="action_open_task"
                     title="Open task"
                     tabindex="-1"
                     type="object"
                     icon="fa-folder-open-o"
-                    attrs="{'invisible': ['|', ('is_task_closed', '=', False), ('task_id', '=', False)]}"
+                    invisible="is_task_closed == False or task_id == False"
```

Comment:
`invisible="not is_task_closed or not task_id"`

Reply:
THNX for review. Changed.